src/compas_fab/backends/ros/backend_features/move_it_forward_kinematics.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from compas.utilities import await_callback
+
+from compas_fab.backends.backend_feature_interfaces import ForwardKinematics
+from compas_fab.backends.ros.backend_features.helpers import validate_response
+from compas_fab.backends.ros.messages import GetPositionFKRequest
+from compas_fab.backends.ros.messages import GetPositionFKResponse
+from compas_fab.backends.ros.messages import Header
+from compas_fab.backends.ros.messages import JointState
+from compas_fab.backends.ros.messages import MultiDOFJointState
+from compas_fab.backends.ros.messages import RobotState
+from compas_fab.backends.ros.service_description import ServiceDescription
+
+__all__ = [
+    'MoveItForwardKinematics',
+]
+
+
+class MoveItForwardKinematics(ForwardKinematics):
+    """Callable to calculate the robot's forward kinematic."""
+    GET_POSITION_FK = ServiceDescription('/compute_fk',
+                                         'GetPositionFK',
+                                         GetPositionFKRequest,
+                                         GetPositionFKResponse,
+                                         validate_response)
+
+    def __init__(self, ros_client):
+        self.ros_client = ros_client
+
+    def forward_kinematics(self, configuration, group=None, options={}):
+        """Calculate the robot's forward kinematic.
+
+        Parameters
+        ----------
+        configuration : :class:`compas_fab.robots.Configuration`
+            The full configuration to calculate the forward kinematic for. If no
+            full configuration is passed, the zero-joint state for the other
+            configurable joints is assumed.
+        group : str, optional
+            The planning group used for the calculation. Defaults to the robot's
+            main planning group.
+        options : dict, optional
+            Dictionary containing the following key-value pairs:
+
+            - backend :: None or str
```

Comment:
A single colon in some instances has some strange behavior. The following in a doc string results in the attached image: 
```
"""
        options: dict, optional
            Dictionary containing the following key-value pairs:

            - base_link (:obj:`str`) : Name of the base link.
            - avoid_collisions : bool, optional
                Whether or not to avoid collisions. Defaults to `True`.
            - constraints :: list of :class:`compas_fab.robots.Constraint`, optional
                A set of constraints that the request must obey. Defaults to `None`.
"""
```

<img width="631" alt="Screenshot 2020-06-08 at 12 03 53" src="https://user-images.githubusercontent.com/57254617/84019429-8d04d180-a981-11ea-8d44-31a4427b59ab.png">

Reply:
uuuuhhhggg